packages/terra-heading/tests/nightwatch/HeadingSize.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import Heading from '../../lib/Heading';
+
+const HeadingSize = () => (
+  <div>
```

Comment:
Add each size might be a nice visual for functional verification--like with the different levels.